code/modules/jobs/job_exp.dm

```diff
@@ -20,6 +20,8 @@ GLOBAL_PROTECT(exp_to_update)
 		return 0
 	var/my_exp = C.calc_exp_type(get_exp_req_type())
 	var/job_requirement = get_exp_req_amount()
+	if(my_exp > exp_max && !is_donator(C)) // If they have too many hours and are not a donator
```

Comment:
This shouldn’t be overwritten by donators by default, it’s an unintuitive exception in case anyone wants to playtime-limit other roles independently of donator status